File: 백준/Bronze/10810. 공 넣기/공 넣기.cc
#include <stdio.h>

int main(void) {
	int n, m, i, j, k, a, s;
	scanf("%d %d", &n, &m);
	int arr[n];
	
	for(s=0; s<n; s++) {
		arr[s] = 0;
	}
	for(a=0; a<m; a++) {
		scanf("%d %d %d", &i, &j, &k);
			for(i=i; i<=j; i++) {
			arr[i-1] = k;
		}	
	}
	for(a=0; a<n; a++)
	printf("%d ", arr[a]);
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner num = new Scanner(System.in);
        int a;

        int n = num.nextInt();
        int m = num.nextInt();

        int[] arr = new int[n];


        for(a=0; a<m; a++) {
            int i = num.nextInt();
            int j = num.nextInt();
            int k = num.nextInt();

            for(i=i; i<=j; i++) {
                arr[i-1] = k;
            }
        }
        for(a=0; a<n; a++) {
            System.out.print(arr[a] + " ");
        }
    }
}